integration/query_element_test.go

```diff
@@ -85,6 +89,180 @@ func TestExistsOperator(t *testing.T) {
 			require.NoError(t, err)
 
 			var actual []bson.D
+
+			err = cursor.All(ctx, &actual)
+			require.NoError(t, err)
+			assert.Equal(t, tc.expectedIDs, collectIDs(t, actual))
+		})
+	}
+}
+
+func TestTypeOperator(t *testing.T) {
+	t.Parallel()
+	// TODO: add this data types to collection "objectId", "decimal", "minKey", "maxKey"
```

Comment:
objectID was added, and I don't think we ever add minKey and maxKey

Reply:
I've added test for objectID, removed minKey and maxKey from TODO.